ArmRobot/Assets/Scripts/ArticulationJointController.cs

```diff
@@ -9,25 +8,77 @@ public class ArticulationJointController : MonoBehaviour
 {
     public RotationDirection rotationState = RotationDirection.None;
     public float speed = 300.0f;
+    public float r;
+
+    private ConfigurableJoint joint;

```

Comment:
This robot already has working articulation joints, no need to add config joints too. Please remove them.